crypto/nrf_cc310_mbedcrypto/include/mbedtls/rsa_alt.h

```diff
@@ -62,6 +63,8 @@ typedef struct
                                       encoding                          */
 #if defined(MBEDTLS_THREADING_C)
     mbedtls_threading_mutex_t mutex;    /*!<  Thread-safety mutex       */
+#else
+    uint8_t dummy[8];
```

Comment:
would be nice with a comment explaining that this is to ensure context size is always the same size, and that `mbedtls_threading_mutex_t` is 8 byte large (I assume)